ext/hash/hash.c

```diff
@@ -512,10 +512,7 @@ static void php_hash_do_hash_hmac(
 	}
 
 	if (isfilename) {
-		if (CHECK_NULL_PATH(data, data_len)) {
-			zend_argument_value_error(2, "must not contain any null bytes");
-			RETURN_THROWS();
-		}
+		ZEND_ASSERT(!CHECK_NULL_PATH(data, data_len));
```

Comment:
This change isn't right: there are other places that call php_hash_do_hash_hmac too and don't use the "p" ZPP type. I prefer to just drop this commit.

Reply:
ACK